LumpNav/LumpNav.py

```diff
@@ -331,26 +330,25 @@ def setupScene(self): #applet specific
     self.cauteryModel_CauteryTip = slicer.util.getNode('CauteryModel')
     if not self.cauteryModel_CauteryTip:
       if (self.parameterNode.GetParameter('TestMode')=='True'):
-          print "Loading stl from local device"
           moduleDirectoryPath = slicer.modules.lumpnav.path.replace('LumpNav.py', '')
           slicer.util.loadModel(qt.QDir.toNativeSeparators(moduleDirectoryPath + '../../../models/temporary/cautery.stl'))
           self.cauteryModel_CauteryTip=slicer.util.getNode(pattern="cautery")
-      else: # change here
+      else: 
           flag = True
           moduleDirectoryPath = slicer.modules.lumpnav.path.replace('LumpNav.py', '')
           if slicer.util.loadModel(qt.QDir.toNativeSeparators(moduleDirectoryPath + '/Resources/CauteryModel.stl')):
+              print "Loading cautery model"

```

Comment:
Would it be better using logging.debug('whatever message') instead of print 'whatever message'?
We usually don't have the python console open, but when we are looking for errors or events that happened in a session, we look at the Slicer log.